src/prpy/planning/base.py

```diff
@@ -70,7 +70,18 @@ def call_planner():
                 try:
                     planner_traj = self.func(instance, cloned_robot,
                                              *args, **kw_args)
+
+                    # Tag the trajectory with the planner and planning method
+                    # used to generate it. We don't overwrite these tags if
+                    # they already exist.
+                    tags = GetTrajectoryTags(planner_traj)
+                    tags.setdefault('planner', instance.__class__.__name__)
+                    tags.setdefault('planning_method', self.func.__name__)
+                    SetTrajectoryTags(planner_traj, tags, append=False)
+
                     return CopyTrajectory(planner_traj, env=env)
+
+                    return copied_traj
```

Comment:
I might be reading this wrong, but it looks like this returns twice and copied_traj is never set to anything.